Task-2.1.5.1.md

```diff
@@ -0,0 +1,55 @@
+ 
+## Workflow for Metaproteomic Data  
+
+### Background    
+The NMDC initiative will enable users in the environmental microbiology community to easily search and access the microbiome (FICUS) data collected within three national laboratories (PNNL, LANL, and LBNL) using a new framework. Currently, the instrument data (e.g. LC-MS/MS analyses), and data analysis results are being managed via independent systems across labs. This framework will make the different datasets and workflows from each sub-area within environmental microbiology available under one canopy.
+
+### Positioning
+
+  The Integrative Omics group at PNNL currently has their own internal data repository (DMS) that contains metadata, results/data files, etc. related to each sample (single organism or group of meta-organisms) analyzed via mass spectrometry for protein identification and quantification. For long term storage and future external accessibility, DMS data is pushed into the MyEMSL data repository. MyEMSL supports different initiatives across multiple divisions at PNNL to achieve and maintain their respective data and metadata sources.
```

Comment:
I'm not sure if this is totally relevant.

Reply:
ok, so do you want to add/amend what's relevant? or you're just suggesting to remove? or you're suggesting that this is beyond what's needed for Meta-proteomics workflow development?